internal/cmd/keys.go

```diff
@@ -111,6 +119,11 @@ $ infra keys add connector
 				return err
 			}
 
+			if options.Quiet {
+				cli.Output(resp.AccessKey)
+				return nil
+			}
```

Comment:
I'm not convinced this needs a quiet option right now. The output is simple enough that any non essential outputs, like the pretext, should be printed to `stderr` while the essential outputs, in this case `resp.AccessKey` should be printed to `stdout`. The examples will remain the same since only stdout will be captured